deb/common/control

```diff
@@ -104,6 +104,7 @@ Homepage: https://github.com/docker/buildx
 Package: docker-compose-plugin
 Priority: optional
 Architecture: linux-any
+Recommends: docker-buildx-plugin
```

Comment:
Silly question; if compose shells out to buildx,  and running as “standalone” (`docker-compose`) does it shell out to `/whatever/path/for /plugins/docker-buildx` (standalone) or `docker buildx` ? Mostly looking at `Recommends: ` in the `docker-compose-plugin` package, and if that (when installed separate from the CLI) would also be able to use it, or if it should depend on `docker-ce-cli` already having the `Recommends:` for buildx

Reply:
compose executes the CLI plugin directly (standalone) based on binary file path retrieved from `manager.GetPlugin`.

Reply:
Gotcha; thanks! Wanted to double-check we wouldn't end up in a case where the plug-in was there, but compose still trying to execute it in a different location.

We should probably look if we can have good ways for the CLI (when compose is executed as plugin) to pass information to the plugin(s), so that we have a single source of truth for resolving plugins, but that's an existing problem.